scripts/sysman/memory.yml

```diff
@@ -132,6 +132,21 @@ members:
       desc: "[out] The total allocatable memory in bytes (can be less than the `physicalSize` member of $s_mem_properties_t)"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Extension properties for Memory State"
+version: "1.7"
+class: $sMemory
+name: $s_mem_ext_state_t
```

Comment:
should be "mem_state_ext_t" ?

Reply:
fixed

Reply:
@wdamon-intel @jandres742 would you prefer `exp` extension here?

Reply:
@aravindksg : yes please.

Reply:
Updated the structure with exp. Also added the new files for adding extension.